rpms/README.md

```diff
@@ -5,53 +5,74 @@ needed to build a set of RPMs that are useful for working with Geotrellis.  If
 you are not intending to alter the composition or versions of the RPM
 packages, then you should be able to use the published binaries described in
 the README at the root of this project and do not need to build these
-artifacts yourself. Specifically:
+artifacts yourself.
 
-  > **Do nothing** if you are intending to deploy to EMR and have access to
-  > pre-built RPMs.  Simply point the EMR scripts to the relevant resources on
-  > S3.
+Built RPMs are publicly available at:
 
-  > **Fetch, do not build** if you have access to an already-built set of
-  > RPMs, and your purpose is to use these RPMs to build a Docker container.
+```
+s3://geotrellis-build-artifacts/rpms/
+```
 
-If you have a special use case, and the pre-built binaries are not sufficient,
-then you may need to build them yourself.
+If you need to load them during an EMR bootstrap, you can use `s3://geotrellis-build-artifacts/rpms/bootstrap.sh`.
```

Comment:
The standard approach presented by this repo is to use the Makefiles and configs in the `emr` directory.  Presenting the bootstrap script alone may cause confusion.

We also have no automatic means to ensure that this bootstrap script on S3 stays up to date, so that should be addressed if it's somehow not sufficient to supply the bootstrap in `emr`.

Reply:
> The standard approach presented by this repo is to use the Makefiles and configs in the emr directory

True. I removed the ref to bootstrap.sh here in order to keep a consistent story.

> We also have no automatic means to ensure that this bootstrap script on S3 stays up to date

Also true. That's outside the scope of this PR, I created a new issue: https://github.com/geotrellis/geotrellis-deployments/issues/6